orttraining/orttraining/test/python/orttraining_test_ortmodule_autograd.py

```diff
@@ -1210,3 +1209,63 @@ def forward(self, model_input):
     assert not can_run
 
     del os.environ["ORTMODULE_SKIPPED_AUTOGRAD_FUNCTIONS"]
+
+
+def test_pythonop_training_mode():
+    def check_pythonop_training_mode(model, is_eval_mode):
+        ## make sure the ort's PythonOp's training_mode is correct
+        if is_eval_mode:
+            onnx_nodes = (
+                model._torch_module._execution_manager._inference_manager._onnx_models.exported_model.graph.node
+            )
+        else:
+            onnx_nodes = model._torch_module._execution_manager._training_manager._onnx_models.exported_model.graph.node
+
+        found_pythonop = False
+        for node in onnx_nodes:
+            if node.op_type == "PythonOp":
+                found_pythonop = True
+                for attr in node.attribute:
+                    if attr.name == "training_mode":
```

Comment:
optional: I would consider using `continue` to avoid deeply nested logic

Reply:
the graph contains only several nodes